app/components/Views/confirmations/components/TransactionReview/index.js

```diff
@@ -344,7 +343,7 @@ class TransactionReview extends PureComponent {
       ),
       external_link_clicked: 'security_alert_support_link',
     };
-    AnalyticsV2.trackEvent(
+    this.props.metrics.trackEvent(
```

Comment:
Destructure metrics on L339
```suggestion
    metrics.trackEvent(
```